docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+data:
```

Comment:
So I just noticed that there isn't a version here.  Is there a reason why?  It seems like most `docker-compose.yml` files I see begin with `version: "3"`.  I think versioning this file would be preferable.  Thoughts?